src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -323,7 +324,7 @@ private void completeSetupMayThrowSstableException() {
         catch (Throwable t)
         {
             JVMStabilityInspector.inspectThrowable(t);
-            logger.warn("Error loading key or row cache", t);
+            logger.warn("Error loading key or row cache", UnsafeArg.of("loadRowAndKeyCacheError", t));
```

Comment:
I _believe_ we don't need to do anything for `Throwable`s... they should already correctly log the `stacktrace` part of the sls logging spec.